halo/app/monitor.go

```diff
@@ -112,7 +118,7 @@ func monitorEVMForever(ctx context.Context, cfg Config, ethCl ethclient.Client)
 }
 
 // monitorEVMOnce monitors the attached omni_evm height, peers, and sync status.
-func monitorEVMOnce(ctx context.Context, ethCl ethclient.Client) error {
+func monitorEVMOnce(ctx context.Context, ethCl ethclient.Client, readiness *ReadyResponse) error {
 	// Best effort monitoring of peer count, since method not available in auth API.
 	peers, err := ethCl.PeerCount(ctx)
 	if ethclient.IsErrMethodNotAvailable(err) { //nolint:revive // Empty block skips error handling below.
```

Comment:
most halo instances will not have EVM RPC defined, so `ethclient.IsErrMethodNotAvailable` will be true, which means you will never set geth peers, which will mean